### ISSUE TITLE
Can someone help me? Stopped working yesterday

### ISSUE THREAD

**Issue by HW73** [mgp25/Snap-API#205]
Hi, looking for someone who can help me get mine setup. Had it working yesterday but for whatever reason it's stopped. Thanks

**Comment by hako** [comment]
Your problem is most likely to do with this issue: https://github.com/mgp25/SC-API/issues/202 and this issue https://github.com/mgp25/SC-API/issues/201
Also see https://github.com/mgp25/SC-API/issues/202#issuecomment-140954578
@mgp25 or @teknogeek feel free to close this as a duplicate.

If your issue is not this, please submit a debug log.

**Comment by HW73** [comment]
@hako Hey, I've just tried the same files I used before on a different server. I get the following error, I know the error 403 means FORBIDDEN.

http://i.imgur.com/ci31JuO.png

Just tried it on another snapchat account and got the same error. Confusing

**Comment by mgp25** [comment]
@HW73 please read #201 and #202.

Please, always before opening a new issue, check the closed ones.

**Comment by HW73** [comment]
Thanks @mgp25, just annoyed

**Comment by mgp25** [comment]
@HW73 be patient, it will be up as soon as possible, we are working on it.

**Comment by liamcottle** [comment]
I fixed the Attestation server a couple hours ago, so it should all be fine now :)

**Comment by HW73** [comment]
@liamcottle, were you able to fix this problem? Thanks for keeping us updated

"Guys, plz chill. The issue is with my servers. Google made some server side changes, and I am yet to fix this issue on my servers."

**Comment by liamcottle** [comment]
I have fixed the Attestation server issue, however your issue isn't caused by my servers or the PHP lib.

Your network has been blocked by Snapchat themselves.

The block should be lifted within 24 hours. (IP blocks)

Try on a different network, or just wait until they release the ban.

**Comment by HW73** [comment]
@liamcottle, thanks! Is there any way you've found to prevent being banned? It might be because I've logged on a few times on a spare device aswell as the server. I'll just move my server to another datacenter as this one seems to not work on any server I try there.

**Comment by liamcottle** [comment]
The main reasons for banning, are if you make a shit ton of requests to the login endpoint from the same IP in a small amount of time, or if you try to login with the wrong password too many times.

Also, if you are doing a timed poll, eg making a request to SC every 5 minutes, it may cause bans too.

I don't really have any pointers for IP bans since I don't make my requests on the servers. I make them on client devices (since I run the Casper Android app)

**Comment by HW73** [comment]
@liamcottle, I just moved servers to another country and still get the "The network you are connected to has been temporatily blocked because of suspicious activity." Error, I logged into the app on my phone and sent a few snaps to test it, I'm not sure how they know. 

Yeah I see what you mean about the excess requests, it's currently set to run the curl command every 5 minutes, I might change it to 10. Thanks for you help again, if I can't get it to work I'll just have to leave it for 24 hours. 

I'll send some bitcoin to your address on the wiki when I next buy some man, you've been a big help.
